charts/orchestrator/templates/rhdh-operator.yaml

```diff
@@ -1,3 +1,22 @@
+{{- $cert := false }}
+{{- $ingress_secret := (lookup "v1" "Secret" "openshift-ingress" "router-certs-default") }}
+{{- if $ingress_secret }}
+{{- $cert = get $ingress_secret.data "tls.crt" }}
+---
+# This secret is the Openshift router public certificate. Backstage cannot
+#access the ingress secret from another namespaces, so we copy it to trust it.
+#It's mostly for developement pourposes and to avoid the
+#NODE_TLS_REJECT_UNAUTHORIZED=0 env variable.
+#Related issue: https://issues.redhat.com/browse/RHIDP-1847
+apiVersion: v1
+kind: Secret
+metadata:
+  name: rhdh-ingress-cert
+  namespace: {{ .Values.rhdhOperator.subscription.namespace }}
+data:
+    cert: {{ $cert }}
```

Comment:
Best to avoid having helm managing a secret for the reasons I mentioned earlier (helm-operator shows the contents of the generated manifests as part of the CR status)

Reply:
well, and how do you manage this?

Reply:
We are currently creating the RHDH secret that contains the credentials via the `hack/setup.sh` script, which needs to be executed before installing the chart/deploying a CR instance of the orchestrator via the operator.

Without more information on how RHDH consumes its configmaps, I can't suggest an alternative option, but it could be one possibility.